python-regression/tests/features/machine1/1_local_snapshots_tests.feature

```diff
@@ -53,7 +53,7 @@ Feature: Test Bootstrapping With LS
     #Default for test is to issue 10324
     When milestone 10324 is issued on "nodeA-m1"
     #Give the node time to finish syncing properly, then make sure that the node is synced to the latest milestone.
-    And we wait "120" second/seconds
+    And we wait "20" second/seconds
```

Comment:
No the failure in buildkite was due to me removing the `scanMilestonesInQueue()` from the main thread. The failure was due to the pruning not starting because the initialized flag wasn't set to true until a new milestone became present. You can tell by looking at the failures in the tests. They were pertaining to a failure of the node to take the localsnapshot on startup and waiting for the first issued milestone. I just pushed a change to re-include that call. Tests should pass again after that with the reduced times still.